### PR TITLE
chore(000): clean test logs to make them possible to read

### PR DESCRIPTION
**Pull request by arkadiuszos4chain**
# Pull Request Checklist

- [x] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/spv-wallet/blob/main/.github/CODE_STANDARDS.md)
- [x] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/spv-wallet/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] 🏠 I tested my changes locally.
- [x] ✅ I have provided tests for my changes.
- [x] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [ ] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

File: config/load.go [hunk 1]
```diff
@@ -45,13 +45,24 @@ func Load(logger *zerolog.Logger) (appConfig *AppConfig, err error) {
...
 		} else {
-			fmt.Printf("loaded config: %s", cfg)
+			logger.Debug().Msgf("loaded config: %s", cfg)
 		}
 	}
 
 	return appConfig, nil
 }
 
+func LoadForTest() (appConfig *AppConfig) {
```
Comment (chris-4chain) [inline config/load.go:55]: Before merging - fix this lint error: `exported function LoadForTest should have comment or be unexported (revive)`

**Review by chris-4chain — APPROVED** [review]

**Comment by chris-4chain** [comment]
Testing 🧪

File: server/server_test.go [hunk 2]
```diff
@@ -41,7 +40,6 @@ func (ts *TestSuite) SetupTest() {
 	ts.BaseSetupTest()
 
 	// Load the router & register routes
-	ts.Router = gin.Default()
 	require.NotNil(ts.T(), ts.Router)
```
Comment (Nazarii-4chain) [inline server/server_test.go:43]: Please also remove this line

File: server/server_test.go [hunk 3]
```diff
@@ -41,7 +40,6 @@ func (ts *TestSuite) SetupTest() {
 	ts.BaseSetupTest()
 
 	// Load the router & register routes
-	ts.Router = gin.Default()
```
Comment (chris-4chain) [inline server/server_test.go:44]: Testing result: `server/server_test.go` - these unit tests are failing 

`runtime error: invalid memory address or nil pointer dereference`.

**Review by chris-4chain — CHANGES_REQUESTED** [review]

File: config/load.go (hunk 1 above)
Comment (arkadiuszos4chain) [reply config/load.go:56]: fixed

File: server/server_test.go (hunk 3 above)
Comment (arkadiuszos4chain) [reply server/server_test.go:44]: Ha, by the way, I found a bug in auth middleware.

Fixed

**Review by wregulski — APPROVED** [review]

File: server/auth/middleware.go [hunk 4]
```diff
@@ -135,6 +135,7 @@ func SignatureMiddleware(appConfig *config.AppConfig, requireSigning, adminRequi
 	return func(c *gin.Context) {
 		if c.Request.Body == nil {
 			c.AbortWithStatusJSON(http.StatusUnauthorized, "missing body")
+			return
```
Comment (chris-4chain) [inline server/auth/middleware.go:138]: Nice 👍

**Review by chris-4chain — APPROVED** [review]

**Pull request merged**